modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/DataSetDisplayTag.java

```diff
@@ -187,6 +188,10 @@ public String getSelectionType() {
 		return _selectionType;
 	}
 
+	public List<SortItem> getSortItems() {
```

Comment:
@luca-pellizzon @dsitu maybe us better use getSortList as patter?

Reply:
I kept it adherent to what we already have, like getBulkActionDropdownItems() for bulkActionDropdownItems